pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -1657,8 +1657,8 @@ def date_parser(dt, time):
     datetimes = np.array(["2013-11-03T19:00:00"] * 3, dtype="datetime64[s]")
     expected = DataFrame(
         data={"rxstatus": ["00E80000"] * 3},
-        index=MultiIndex.from_tuples(
-            [(datetimes[0], 126), (datetimes[1], 23), (datetimes[2], 13)],
+        index=MultiIndex.from_arrays(
+            [datetimes, [126, 23, 13]],
```

Comment:
why does this need to change?

Reply:
with from_tuples it reconstructs a M8[ns] level.  with from_arrays it keeps the reso `datetimes` already has